### ISSUE TITLE
Temperature missing with some ssd

### ISSUE THREAD

**Issue by Luky2063** [dougiteixeira/proxmoxve#233]
### Checklist

- [X] I have updated to the latest available Proxmox VE Custom Integration version, [see the latest version here](https://github.com/dougiteixeira/proxmoxve/releases/latest).

### Describe the issue you are experiencing

I have installed the lastest available version 3.3.0).
It seems that on some ssd, the smart value for the temperature is not on the attribute "Temperature_Celsius" or "Temperature" but on "Airflow_Temperature_Cel".
In my case, the SSD Samsung 860 PRO 512...

I found that "coordinator.py" (line 589) have these attributes : 
`
                    if disk_attribute["name"] in ("Power_Cycle_Count", "Power Cycles"):
                        disk_attributes["power_cycles"] = disk_attribute["raw"]
                    elif disk_attribute["name"] in (
                        "Temperature_Celsius",
                        "Temperature",
                        "Airflow_Temperature_Cel",
                    ):
`
I fixed it and it worked for me.

Please, could you fix this line.

Thanks a lot for your great work !!

### In which version of Home Assistant Core do you have the problem?

2014.1.2

### What version of Proxmox VE Custom Integration has the issue?

3.3.0

### What version of Proxmox VE do you have the problem?

3.3.0

**Comment by Luky2063** [comment]
Missing log file : 
[proxmoxve-2efc63c11fdd5ad8d632cb6f0cb6843f-Disk Proxmox1__dev_sde-8691d75c9d14e46dd09a2ff541071890.json.txt](https://github.com/dougiteixeira/proxmoxve/files/13851830/proxmoxve-2efc63c11fdd5ad8d632cb6f0cb6843f-Disk.Proxmox1__dev_sde-8691d75c9d14e46dd09a2ff541071890.json.txt)

**Comment by dougiteixeira** [comment]
It may seem like it, but the "Airflow Temperature" attribute does not represent the disk temperature, but this does not always apply.

Some manufacturer may set a different value, [such as reference](https://en.wikipedia.org/wiki/Self-Monitoring,_Analysis_and_Reporting_Technology#Known_ATA_S.M.A.R.T._attributes):
```
Value is equal to (100-temp. °C), allowing manufacturer to set a minimum threshold which corresponds to a maximum temperature. This also follows the convention of 100 being a best-case value and lower values being undesirable. However, some older drives may instead report raw Temperature (identical to 0xC2) or Temperature minus 50 here.
```

In your case, I believe it is the disk temperature, [according to this document](https://download.semiconductor.samsung.com/resources/others/SSD_Application_Note_SMART_final.pdf):
```
The current temperature of the area surrounding the NAND chips
inside of the SSD.
```

So I implemented the additional sensor in #239.
